### ISSUE TITLE
[a11y] Menu button doesn't work as expected with overlay shown

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#730]
STR
- activate voiceview
- Go to page, open overlay (double-tap menu)
- Double-tap menu to close menu

Actual: Sometimes it closes the overlay, sometimes it reopens it
Expected: Overlay is closed

I think we need to only run our special menu button handling (to close on menu press) if VoiceView is disabled and that'd just fix this issue.
